### PR TITLE
BUILD-92: update WaitForOpenShiftNamespaceImageStreams with more detailed diagnostics

### PR DESCRIPTION
**Pull request by gabemontero**
Surface detailed error analysis in final error reported by ginkgo in the e2e's for waiting on the sample imagestreams in openshift

include direction on how to report possible outages with registry.redhat.io

### CODE REVIEW

**Comment by gabemontero** [comment]
ok only the recently flake e2e-aws-serial suite failing 

/assign @bparees 

I believe you asked about improving the debug assistance around TBR outages a while back ... if recollection serves, here is my take on how to do that using the e2e infra I previously introduced ... if you feel inclined to approve, etc. while @adambkaplan is out, please do.

otherwise, unassign as you see fit, and 

/assgin @adambkaplan 
/assign @coreydaley 

thanks

File: test/extended/util/framework.go [hunk 1]
```diff
@@ -326,18 +323,32 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
...
 		e2e.Logf("Success! \n")
 		return nil
 	}
 	DumpImageStreams(oc)
 	DumpSampleOperator(oc)
-	return fmt.Errorf("Failed to import expected imagestreams")
+	errorString := ""
+	if strings.Contains(errorString, "FailedImageImports") {
+		strbuf := bytes.Buffer{}
+		strbuf.WriteString(fmt.Sprintf("Issues exist pullilng images from registry.redhat.io: %s\n", scanErr.Error()))
```
Comment (bparees) [inline test/extended/util/framework.go:343]:
```suggestion
		strbuf.WriteString(fmt.Sprintf("Issues exist pulling images from registry.redhat.io: %s\n", scanErr.Error()))
```

File: test/extended/util/framework.go [hunk 2]
```diff
@@ -326,18 +323,32 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
...
-	if success {
+	if pollErr == nil {
 		e2e.Logf("Success! \n")
 		return nil
 	}
 	DumpImageStreams(oc)
 	DumpSampleOperator(oc)
-	return fmt.Errorf("Failed to import expected imagestreams")
+	errorString := ""
+	if strings.Contains(errorString, "FailedImageImports") {
```
Comment (bparees) [inline test/extended/util/framework.go:341]: doesn't look like you're setting errorString correctly here

Comment (gabemontero) [reply test/extended/util/framework.go:341]: ahhh ... classic last minute change caught me ... thanks

**Comment by gabemontero** [comment]
updates pushed @bparees - thanks

**Pull request merged**